test-commons/src/main/java/com/sonarsource/checks/verifier/internal/InternalIssueVerifier.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * SonarQube Analyzer Test Commons
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonarsource.checks.verifier.internal;
+
+import com.sonarsource.checks.verifier.FileContent;
+import com.sonarsource.checks.verifier.MultiFileVerifier;
+import com.sonarsource.checks.verifier.SingleFileVerifier;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.nio.charset.StandardCharsets'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)